### ISSUE TITLE
A few questions about commented codes.

### ISSUE THREAD

**Issue by lin01109** [nicholassung97/Neuroevolution-of-PINNs#1]
HI，I'm a postgraduate student and recently interested in this project, but there are a few codes which confuse me and it would be a great pleasure if you could explain it to me. 

In code/Nonlinear/non_linear_cma.jax.ipynb, I notice that a few lines are commented out when defining reset_fn and step_fn in Class linear(VectorizedTask).

**reset_fn**

        def reset_fn(key):
            batch_data, batch_labels = data_pde, labels_pde
            # if test:
            #     # In the test mode, we want to test on the entire test set.
            #     batch_data, batch_labels = data_pde, labels_pde
            # else:
            #     # In the training mode, we only sample a batch of training data.
            #     batch_data, batch_labels = sample_batch(
            #         key, data_pde, labels_pde, data_bc, labels_bc, batch_size)
            return State(obs=batch_data, labels=batch_labels)

**step_fn**

        def step_fn(state, action):
            # if test:
            #     # In the test mode, we report the mean square error
            #     "change to training loss"
            #     # reward = loss(action, state.obs, state.labels)
            #     reward = mse(action)
            # else:
                # In the training mode, we return the negative loss as the
                # fitness. 
                # the action (prediction) will include u, u_x and u_xx
            reward = -loss(action, state.obs)
            # This is an one-step task, so that last return value (the `done`
            # flag) is one.
            
            return state, reward, jnp.ones(())

It seems that they involve switching between training modes.I would like to know that when I run code if they should be uncommented out. Besides, I tried to uncomment lines in reset_fn(key) and it seemed like that function sample_batch() wasn't be defined. 

Thanks a lot!!!!!Good luck with scientific research!!!
